### ISSUE TITLE
Problem in log_validation of train_control.py

### ISSUE THREAD

**Issue by laulampaul** [aigc-apps/CogVideoX-Fun#53]
when I try to run train_control.py , there is a problem in log_validation 
In README_TRAIN_CONTROL.md , there is no args.train_mode
but in the func log_validation

for i in range(len(args.validation_prompts)):
            with torch.no_grad():
                if args.train_mode != "normal":

how can I revise the code and set args.train_mode ?

**Comment by bubbliiiing** [comment]
export MODEL_NAME="models/Diffusion_Transformer/CogVideoX-Fun-2b-InP"
export DATASET_NAME="datasets/internal_datasets/"
export DATASET_META_NAME="datasets/internal_datasets/metadata.json"
export NCCL_IB_DISABLE=1
export NCCL_P2P_DISABLE=1
NCCL_DEBUG=INFO

# When train model with multi machines, use "--config_file accelerate.yaml" instead of "--mixed_precision='bf16'".
accelerate launch --mixed_precision="bf16" scripts/train.py \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --train_data_dir=$DATASET_NAME \
  --train_data_meta=$DATASET_META_NAME \
  --image_sample_size=1024 \
  --video_sample_size=256 \
  --token_sample_size=1024 \
  --video_sample_stride=3 \
  --video_sample_n_frames=49 \
  --train_batch_size=1 \
  --video_repeat=1 \
  --gradient_accumulation_steps=1 \
  --dataloader_num_workers=8 \
  --num_train_epochs=100 \
  --checkpointing_steps=50 \
  --learning_rate=2e-05 \
  --lr_scheduler="constant_with_warmup" \
  --lr_warmup_steps=100 \
  --seed=42 \
  --output_dir="output_dir" \
  --gradient_checkpointing \
  --mixed_precision="bf16" \
  --adam_weight_decay=3e-2 \
  --adam_epsilon=1e-10 \
  --vae_mini_batch=1 \
  --max_grad_norm=0.05 \
  --random_hw_adapt \
  --training_with_video_token_length \
  --random_frame_crop \
  --enable_bucket \
  --use_came \
  --train_mode="inpaint" \
  --resume_from_checkpoint="latest" \
  --trainable_modules "."

**Comment by bubbliiiing** [comment]
Sorry, we haven't fully tested it yet. We need to fix it
